search/types_skipper.cpp

```diff
@@ -64,17 +74,30 @@ bool TypesSkipper::SkipAlways(feature::TypesHolder const & types) const
 {
   for (auto type : types)
   {
-    ftype::TruncValue(type, 2);
-    if (HasType(m_skipAlways[1], type))
-      return true;
-
     ftype::TruncValue(type, 1);
     if (HasType(m_skipAlways[0], type))
       return true;
   }
   return false;
 }
 
+bool TypesSkipper::SkipSpecialNames(feature::TypesHolder const & types, std::string_view defName) const
+{
+  // Sinse we assign ref tag into name for entrances, this is a crutch to avoid indexing
```

Comment:
```suggestion
  // Since we assign ref tag into name for entrances, this is a crutch to avoid indexing
```

Reply:
maybe leave a TODO if you want:
// TODO : a better solution would be to store entrance refs in metadata